### PR TITLE
global: docs and installation fixes

### PR DESCRIPTION
**Pull request by alizeepace**

### CODE REVIEW

**Comment by tiborsimko** [comment]
@jirikuncar @lnielsen Can you please switch off Coveralls notifications for `invenio-metrics`? I don't seem to have right for this particular repository...

File: setup.py [hunk 1]
```diff
@@ -118,9 +119,9 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
```
Comment (tiborsimko) [inline setup.py]: Please remove the PyPy classifier as we don't test for pypy during CI yet.

File: setup.py [hunk 2]
```diff
@@ -118,9 +119,9 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
```
Comment (tiborsimko) [inline setup.py:122]: Please remove 3.3 and 3.4 also from `.travis.yml`.

**Comment by lnielsen** [comment]
@tiborsimko Fixed. It doesn't make sense to fix anything in this repo. Code in it is outdated, so I'd rather remove the repo until we have had time to fix it.

**Comment by tiborsimko** [comment]
@lnielsen OK.  Since the PR is basically ready, I'll merge it anyway :smile: but let's consider moving this repository away to an appropriate "incubating" organisation...

**Pull request merged**